CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ All notable changes to `dash-ag-grid` will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 Links "DE#nnn" prior to version 2.0 point to the Dash Enterprise closed-source Dash AG Grid repo
 
+## [unreleased]
+
+### Removed
+
+### Added
+
+### Updated
+- [#174](https://github.com/plotly/dash-ag-grid/pull/174)
+  - `columnState` floats during grid interaction and only gets pushed when sent in a callback
+  - `columnDefs` trumps `columnState` if it is pushed in a callback without a `columnState`
+
+### Fixed
+- [#174](https://github.com/plotly/dash-ag-grid/pull/174)
+  - `Markdown` renderer now will pass back `''` instead of `undefined` if there isnt a value
```

Comment:
it passes back `null`, not `''` - but that's an implementation detail, what we want to describe here is the effect of this for users, perhaps:
```suggestion
  - `Markdown` renderer now displays a blank cell rather than writing `undefined` if there is no value. Fixes [#171](https://github.com/plotly/dash-ag-grid/issues/171)
```